moveit_ros/moveit_servo/src/cpp_interface_demo/servo_cpp_interface_demo.cpp

```diff
@@ -186,8 +187,12 @@ int main(int argc, char** argv)
 
   // Spin
   auto executor = std::make_unique<rclcpp::executors::MultiThreadedExecutor>();
-  executor->add_node(node);
-  executor->spin();
+  rclcpp::Rate loop_rate(DEFAULT_SPIN_RATE);
+  while (rclcpp::ok())
+  {
+    executor->spin_node_once(node);
```

Comment:
Why not use spin_some()?

Reply:
I'm not aware of the pros/cons of that. Do you have any reason to suggest it? I will try it.

Reply:
I find that replacing `executor->spin_once(node);` with `executor->spin_some();` does not work -- the robot doesn't move and CPU load is just as high.

Reply:
Oh, it does work if I do it right. Seems to be functionally equivalent to spin_once() as far as I can tell

Reply:
I think the advantage of spin_some() will be when you have multiple nodes on an executor (and possibly some small gains from not adding the node to the executor and then removing it at the end of each call to spin once).